lib/rubocop/cop/style/redundant_parentheses.rb

```diff
@@ -126,8 +128,17 @@ def only_closing_paren_before_comma?(node)
           line_range.source =~ /^\s*\)\s*,/
         end
 
-        def disallowed_literal?(node)
-          node.literal? && !ALLOWED_LITERALS.include?(node.type)
+        def disallowed_literal?(begin_node, node)
+          node.literal? &&
+            !ALLOWED_LITERALS.include?(node.type) &&
+            !raised_to_power_negative_numeric?(begin_node, node)
+        end
+
+        def raised_to_power_negative_numeric?(begin_node, node)
+          return false unless node.int_type? || node.float_type?
+          return false if node.children.first >= 0 || begin_node.parent.nil?
```

Comment:
node.children.first >= 0? I don't immediately understand what you're checking for here. I'm guessing it's whether  the base number is positive of negative but it's really hard to infer from the code you've written.